<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="play">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="atras">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="adelante">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>350</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addSong">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>420</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nombreCancion">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre canci√≥n</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="progreso">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="quitar">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>420</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cancionesCargadas">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>500</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Canciones cargadas</string>
    </property>
   </widget>
   <widget class="QLabel" name="tActual">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>T.Act</string>
    </property>
   </widget>
   <widget class="QLabel" name="tTotal">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>290</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>T.Tot</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listaCanciones">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>231</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="aux">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="aleatorio">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="loop">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>420</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelCovers">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>127</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>play</zorder>
   <zorder>atras</zorder>
   <zorder>adelante</zorder>
   <zorder>addSong</zorder>
   <zorder>nombreCancion</zorder>
   <zorder>progreso</zorder>
   <zorder>quitar</zorder>
   <zorder>cancionesCargadas</zorder>
   <zorder>tActual</zorder>
   <zorder>tTotal</zorder>
   <zorder>aux</zorder>
   <zorder>listaCanciones</zorder>
   <zorder>aleatorio</zorder>
   <zorder>loop</zorder>
   <zorder>labelCovers</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>737</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
